package.json

```diff
@@ -234,6 +234,7 @@
     "@babel/runtime@npm:^7.18.3": "patch:@babel/runtime@npm%3A7.24.0#~/.yarn/patches/@babel-runtime-npm-7.24.0-7eb1dd11a2.patch",
     "@babel/runtime@npm:^7.8.3": "patch:@babel/runtime@npm%3A7.24.0#~/.yarn/patches/@babel-runtime-npm-7.24.0-7eb1dd11a2.patch",
     "@babel/runtime@npm:^7.8.4": "patch:@babel/runtime@npm%3A7.24.0#~/.yarn/patches/@babel-runtime-npm-7.24.0-7eb1dd11a2.patch",
+    "@metamask/controller-utils@npm:^8.0.4": "9.1.0",
```

Comment:
thanks for the comment! Good call. Updated the PR to only resolve @spruceid/siwe-parser rather than @metamask/controller-utils to minimize changes https://github.com/MetaMask/metamask-extension/pull/24138/commits/3fc4ff3325edcde7cb77a0f63ceb4abfe4c3d3f9

I think there's decent unit test coverage for signature pages related to SIWE including the [new ones](https://github.com/MetaMask/core/issues/4141) you created for @metamask/controller-utils. We should add more e2e tests to cover message variations: with and without URL schemas, EIP-55 and non-EIP-55, test-dapp buttons, and more. This said, we don't have the bandwidth for this now and will be working on e2e tests for redesigned pages in the near future. They will be covered here: https://github.com/MetaMask/metamask-extension/issues/23977